### PR TITLE
Remove UpdateCommand interface from command store

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

Just as we do on PR #3201 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/datastore/commandstore.go [hunk 1]
```diff
@@ -34,22 +34,12 @@ func (c *commandCollection) Factory() Factory {
...
-		}
-	}
-)
-
 type CommandStore interface {
 	Add(ctx context.Context, cmd *model.Command) error
 	Get(ctx context.Context, id string) (*model.Command, error)
 	List(ctx context.Context, opts ListOptions) ([]*model.Command, error)
-	UpdateCommand(ctx context.Context, id string, updater func(piped *model.Command) error) error
+	MarkAsTimeOut(ctx context.Context, id string) error
```
Comment (nghialv) [inline pkg/datastore/commandstore.go:41]: `UpdateStatus` to be unified.

File: pkg/datastore/commandstore.go [hunk 2]
```diff
@@ -120,3 +110,19 @@ func (s *commandStore) UpdateCommand(ctx context.Context, id string, updater fun
 		return p.Validate()
 	})
 }
+
+func (s *commandStore) MarkAsTimeOut(ctx context.Context, id string) error {
+	return s.update(ctx, id, func(c *model.Command) error {
+		c.Status = model.CommandStatus_COMMAND_TIMEOUT
+		return nil
+	})
```
Comment (nghialv) [inline pkg/datastore/commandstore.go:118]: Add a private `update` function to ensure that `UpdatedAt` field is being updated.

File: pkg/datastore/commandstore.go (hunk 1 above)
Comment (khanhtc1202) [reply pkg/datastore/commandstore.go:41]: I feel you 👍 But there is one thing that concerns me, this `MarkAsTimeout` interface only be used to make the status of the command entity to `CommandStatus_COMMAND_TIMEOUT`, not the other kind. Just realized that this `c.Status` field be touched by both `ops` and `piped` at the same time, it's time for us to move this orphancommandcleaner job to other place than ops 🤔

File: pkg/datastore/commandstore.go (hunk 2 above)
Comment (khanhtc1202) [reply pkg/datastore/commandstore.go]: yeh, it's 👍

File: pkg/datastore/commandstore.go (hunk 1 above)
Comment (khanhtc1202) [reply pkg/datastore/commandstore.go]: I get your point, just remove this `MarkAsTimeout` and use only once `UpdateStatus` interface in all cases. PTAL when you have time.

**Review by nghialv — APPROVED** [review]

**Pull request merged**